code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -93,6 +99,38 @@ GLOBAL_LIST_INIT(iron_recipes, list ( \
 	. = ..()
 	recipes = GLOB.sandstone_recipes
 
+// i know there's "sand" in mining module (aka impure silicates aka glass ore), but it doesn't work as a sheet and has some other problems
+// you can remove this if you turn all ores into sheets so they stack, or maybe i'll rework mining module myself
```

Comment:
What? Just replace all instances of this old sand type with your new one and delete this comment.